### ISSUE TITLE
Sign-in widget change from sitewide content team

### ISSUE THREAD

**Issue by katherine-fung** [department-of-veterans-affairs/va.gov-team#58433]
### Indicate if you need front-end engineering support for

- [ ] Content audit
- [X] Implementation of REACT sign-in widgets
- [ ] Support for UAT testing or phased roll out of app/tool landing page
- [ ] Something else

### Describe your request

Now that the product team has started using the term "decision reviews" in the claim status tool, we need to add the phrase to the header of the sign-in widget on the [landing page](https://www.va.gov/claim-or-appeal-status/). 

The header should say: 

"Sign in to see your claim, decision review, or appeal status" 

Can your team help make this change? 

### Time constraints / launch deadlines

ASAP - decision reviews are already available in the claims status tool. We are waiting on this widget change, before we can publish the rest of the edits to the landing page

### Your team name

sitewide content

### Your product/team Slack channel

sitewide-content-ia

### Your DEPO product owner

Beth Potts

### Notify Sitewide Public Websites of this issue, and tag @Wes Rowe & @jilladams.

- [X] I acknowledge that I must notify `#sitewide-public-websites` in Slack after submitting this issue.

**Comment by jilladams** [comment]
Hi folks - Public websites doesn't own the widget on the Claims Status page: https://github.com/department-of-veterans-affairs/vets-website/blob/main/.github/CODEOWNERS#LL53C1-L53C1

That belongs to `@department-of-veterans-affairs/benefits-team-1-frontend` - https://github.com/orgs/department-of-veterans-affairs/teams/benefits-team-1-frontend/members

@Mottie @jaredcunha  are you able to help us get this ticket moved to the right place for your team to take action?

**Comment by jacobworrell** [comment]
@jilladams seems like we're going in circles a bit here. @katherine-fung originally made the request to change the language in the widget header.  @jaredcunha is on my team and our team is the one kicking this back to @katherine-fung now. 

The reason we wouldn't meet this request is because we can't find where that widget lives in the code base. It's not actually part of the CST application, it's something separate. We're happy to make the change but we haven't been able to find where the code lives to make the change. 

Sorry for the frustration. Do you know who could help us locate where this code lives? 

cc @jerekshoe @samcoforma @kjduensing

**Comment by Mottie** [comment]
@jerekshoe and I discussed this recently

The [CST call-to-action widget definition](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/static-pages/cta-widget/ctaWidgets.js#L70-L81) would need to be updated.

And to fix the heading level would require changing this code:

Change [this line](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/static-pages/cta-widget/index.js#L152)

to this:

```js
headerLevel={ctaWidget?.headerLevel || this.props.headerLevel}
```

then in the CST definition, we would add a `headerLevel` to the appropriate value.

**Comment by jilladams** [comment]
Benefits team has a PR in flight. Public Websites will get tagged as codeowners and need to review. (FYI @jtmst @dsasser @randimays ).  Context: https://dsva.slack.com/archives/C52CL1PKQ/p1684170295088959

**Comment by jerekshoe** [comment]
@jilladams The PR associated with this ticket has been merged. I'm thinking this is safe to close now; This is what I am seeing on the CST landing page:
<img width="714" alt="Screenshot 2023-05-22 at 1 28 20 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/13838621/8f050e2c-7f41-4e6e-b158-21e5ce318427">

**Comment by jerekshoe** [comment]
@katherine-fung was there something I missed on this one?

**Comment by katherine-fung** [comment]
Hi @jerekshoe @jacobworrell @jilladams @Mottie, 

Thank you so much for locating and updating the code. 

First, I apologize - this is my oversight. I meant to ask you to replace the word "see" with "check" for inclusivity. Could you please update the copy to say the words in quotes? 

"Sign in to check your claim, decision review, or appeal status"

Second, the copy "Sign in to... appeal status" was previously a header with header styling. I see that the first line of the sign-in widget copy has lost its header styling on this page (and on other pages. For example: [secure messaging](https://www.va.gov/health-care/secure-messaging/), [download VA letters](https://www.va.gov/records/download-va-letters/), [change your direct deposit](https://www.va.gov/change-direct-deposit/)). 



Could you please change the first lines of these widgets to H3s?

**Comment by sara-amanda** [comment]
## CAIA Feedback
[**5/22/2023**: The full discussion can be found in this CAIA Slack thread, for reference.](https://oddball.slack.com/archives/C04JVQGEK8F/p1684783039516799)

### Heading is Undefined

To have this read as a heading level, you would need it to look like this:
`<h2 slot="headline"> not what is currently at: <hundefined slot="headline">`

### Recommendations

- [ ] Use proper heading levels and nesting.
- [ ] Inquire to see if this can dynamically change to match the heading hierarchy, yet maintain one styled appearance - or adopt one constant style and adjust heading hierarchy on the page accordingly.
- [ ] Refer to reference items, below, to verify there are no other missing or incorrect items in the code.


### Reference Items

- [ ] [Reference Alert Component from the DST](https://design.va.gov/components/alert#accessibility-considerations)
- [ ] [Reference the Storybook code](https://design.va.gov/storybook/?path=/story/components-va-alert--sign-in-to-start-your-application)
- [ ] Verify there are no other missing or incorrect items in the code. 

#### Current HTML
_This is the HTML was in there currently as of 5/22/2023:_

`<va-alert visible="true" status="continue" class="hydrated"><hundefined slot="headline">Sign in to send secure messages</hundefined><div><p>Sign in with your existing <strong>Login.gov</strong>, <strong>ID.me</strong>, <strong>DS Logon</strong>, or <strong>My HealtheVet</strong> account. If you don't have any of these accounts, you can create a free <strong>Login.gov</strong> or <strong>ID.me</strong> account now.</p><button class="va-button-primary">Sign in or create an account</button></div></va-alert>`

#### Storybook HTML

_This is what Storybook has in the code for an alert with a button:_
```
<va-alert
  close-btn-aria-label="Close notification"
  status="info"
  visible
>
  <h2 slot="headline">
    Sign in now to save time and save your work in progress
  </h2>
```

cc: @katherine-fung @laurwill @SarahKay8 @coforma-terry

**Comment by jerekshoe** [comment]
Just for some context, this is broken because of these changes that were merged on May 9th: https://github.com/department-of-veterans-affairs/vets-website/pull/24042/files#diff-d5b65c152d1c2f1423699e682c6dc0a8a0ce7b79671804092db5201fb0e8c878R17-R20. @jacobworrell how do you want to proceed with this ticket?

**Comment by jilladams** [comment]
@jtmst for awareness in case we can lend any insight based on the react component migration change. @jerekshoe can you say more about what you're running into as a result of that change?

**Comment by jerekshoe** [comment]
> @jtmst for awareness in case we can lend any insight based on the react component migration change. @jerekshoe can you say more about what you're running into as a result of that change?

Because of that change we are seeing all CTA widget components have headers with a tag of `hundefined`. The old `AlertBox` react component used a default header level of 3 if you didn’t specify the `level` prop, but the `va-alert` doesn’t work that way and you have to configure the heading tag manually in the components that use it. The `CallToActionAlert` component should probably set a default value for the `headerLevel` prop so that it always has a valid `h` tag

**Comment by laurwill** [comment]
Hi @jilladams and @jerekshoe , just want to ask if a solution is possible that would set these header levels according to the previous header on the page — these currently appear under H1 on some pages, H2 on others. So the correct header level changes page to page. 

Totally understand if that's not possible right now. Setting them all to default to H3 would go back to the status quo before this change, and we can work on fixing hierarchy page by page.

**Comment by jilladams** [comment]
I agree --  it'll likely take some effort bc reading / setting header levels in context has been tricky, so i wouldn't want to leave us without any levels meantime. I'll make note to the team, and we can ticket a more holistic solution to get prioritized. (FYI @wesrowe )

@jerekshoe are you all able to handle setting things back to H3, or should PW really own that, as a follow up on the React migration work?

**Comment by jerekshoe** [comment]
@jilladams I'm able to set them to h3's, but it might be worth having your team determine the best way to manage what the header level is across pages, especially if it varies from page to page

**Comment by jilladams** [comment]
Yep, makes sense. Discussion with our team here; https://dsva.slack.com/archives/C52CL1PKQ/p1685128804266039

**Comment by sara-amanda** [comment]
@RLHecht adding sitewide CAIA label to this ticket

**Comment by katherine-fung** [comment]
Hi @jilladams and @jerekshoe, I just want to follow up. Are we making the copy on the [claim status tool page](https://www.va.gov/claim-or-appeal-status/) an H3, or are we working on a broader solution where we set the header level on copy across multiple widgets/pages? 

In any case, @jerekshoe, could you change the copy in the widget to: Sign in to check your claim, decision review, or appeal status

(sorry, I mistakenly requested "see" instead of "check" in my original request)

**Comment by jilladams** [comment]
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13860 tracks the Public Websites team intention to  update CTA component to default to H3 when a heading level is not passed in. We think we'll be able to knock this out in our next sprint, starting Wed 6/7.

**Comment by jerekshoe** [comment]
Update: The Claim Status CTA Widget has been update to say "Sign in to check your claim, decision review, or appeal status"
